reveal/run.js

```diff
@@ -107,7 +107,16 @@ eslint no-param-reassign: [
         { src: 'reveal/plugins/zenika-footer/zenika-footer.js', condition() { return !!document.body.classList; } },
         { src: `${config.revealModule}plugin/math/math.js`, async: true },
       ],
-    });
+    };
+
+
+    fileExists('/framework/index.js')
```

Comment:
Isn't this running in the browser? Where is this function coming from? I'm confused.

Reply:
This is a function defined at the bottom of this file. 
https://github.com/Zenika/zenika-formation-framework/blob/d1bb158a5d92cd603b87b1cb1d2632c99868930d/reveal/run.js#L258-L276

Reply:
Oh OK ^^'